src/Tickets/Commerce/Status/Pending.php

```diff
@@ -60,8 +60,31 @@ public function get_name() {
 	 * {@inheritdoc}
 	 */
 	public function can_apply_to( $order, $new_status ) {
-		$status = parent::can_apply_to( $order, $new_status );
+		return $this->can_apply_to_pending_status( parent::can_apply_to( $order, $new_status ), $order );
+	}
 
+	/**
+	 * Whether a Status Interface can be applied on top of another Status Interface.
+	 *
+	 * @since TBD
+	 *
+	 * @param self $current_status The current status.
+	 * @param self $new_status     The new status.
+	 * @param ?int $order_id       Which order we are testing against.
+	 *
+	 * @return bool|WP_Error
+	 */
+	public function status_can_apply_to_status( $current_status, $new_status, $order_id = null ) {
+		return $this->can_apply_to_pending_status( parent::status_can_apply_to_status( $current_status, $new_status, $order_id ), $order_id );
```

Comment:
Very long line: break it down by collecting the first return value in a variable or by breaking lines:
```suggestion
		return $this->can_apply_to_pending_status(
		    parent::status_can_apply_to_status( $current_status, $new_status, $order_id ), 
		    $order_id 
		);
```